doc/appendix-de-kassensichv/receipt-case-definitions/vouchers.md

```diff
@@ -1,28 +1,42 @@
-### Voucher (goods/services)
+# Voucher (goods/services)
 
-The issuance and redemption of vouchers are processes with DSFinV-K requirements. According to the [EU directive](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX%3A32016L1065) as regards the treatment of vouchers there are two kinds of vouchers to be considered:
+The issuance and redemption of vouchers are processes with DSFinV-K requirements. According to the [EU directive](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX%3A32016L1065) as regards the treatment of vouchers, there are two kinds of vouchers to be considered:
 
 - Single-purpose voucher (DE: Einzweckgutscheine)
 - Multi-purpose voucher (DE: Mehrzweckgutscheine)
 
-The difference is, that for single-purpose vouchers, the VAT rate is known already at transfer (issue) because it refers to a single purpose (service or good). On the other hand, in case of multi purpose vouchers the VAT rate is not known because it can be redeemed for different purposes (goods or services). A multi-purpose voucher is a money substitute.
+The difference is that for single-purpose vouchers, the VAT rate is known already at transfer (issue) because it refers to a single purpose (service or good). On the other hand, in case of multi-purpose vouchers, the VAT rate is unknown because it can be redeemed for different purposes (goods or services). A multi-purpose voucher is a money substitute.
+
+## Single-purpose voucher handling
 
 The issuance of single-purpose vouchers is covered by our charge item cases (`ftChargeItemCase`): `0x4445000000000061`, `0x4445000000000062`, `0x4445000000000063`, `0x4445000000000064`, `0x4445000000000065`, `0x4445000000000066` and `0x4445000000000067`.
 
 The redemption of single-purpose vouchers is covered by `ftChargeItemCase`: `0x4445000000000069`, `0x444500000000006A`, `0x444500000000006B`, `0x444500000000006C`, `0x444500000000006D`, `0x444500000000006E`, `0x444500000000006F`. When used in the request, the charge item amount of the good or service should be a positive value and the charge item amount of the voucher should be negative (e.g. +15 and -15).
 
-If the cash register can not handle the redemption of single-purpose vouchers within the charge items than it can alternatively use a pay item having the pay item case (`ftPayItemCase`): `0x444500000000000A`. When used like this in the request, the charge item amount of the good or service should be a positive value and the pay item amount of the voucher should also be a positive value (e.g. +15 and +15).
+### Alternatives
+
+If the cash register can not handle the redemption of single-purpose vouchers within the charge items than it can alternatively use a pay item having the pay item case (`ftPayItemCase`): `0x444500000000000A`. When used like this in the request, the charge item amount of the good or service should be a positive value, and the pay item amount of the voucher should also be a positive value (e.g. +15 and +15).
```

Comment:
..within the charge items, THEN it can alternatively (i suppose this is what you meant here!)

Reply:
jepp!